src/components/SingleArticleCard.jsx

```diff
@@ -0,0 +1,90 @@
+import React from "react";
+import Card from "react-bootstrap/Card";
+import ListGroup from "react-bootstrap/ListGroup";
+import Stack from "react-bootstrap/Stack";
+import Placeholder from "react-bootstrap/Placeholder";
+import { useState, useEffect } from "react";
+import { useParams, Link } from "react-router-dom";
+import { getDay, getHour } from "../Utils/utils";
+
+const SingleArticleCard = () => {
+	const [singleArticle, setSingleArticle] = useState({});
+	const [loading, setLoading] = useState(true);
+	const { article_id } = useParams();
+	const { id, author, body, comment_count, created_at, title, topic, votes } =
+		singleArticle;
+
+	useEffect(() => {
+		const url = `https://caragea-nc-news-backend.herokuapp.com/api/articles/${article_id}`;
+		fetch(url)
+			.then((response) => response.json())
+			.then(({ article }) => {
+				setSingleArticle(article);
+				setLoading(false);
+			});
+	}, [article_id]);
+
+	if (loading)
+		return (
+			<div className="d-flex justify-content-around">
+				<Card style={{ width: "18rem" }}>
+					<Card.Body>
+						<Placeholder as={Card.Title} animation="glow">
+							<Placeholder xs={6} />
+						</Placeholder>
+						<Placeholder as={Card.Text} animation="glow">
+							<Placeholder xs={7} /> <Placeholder xs={4} />{" "}
+							<Placeholder xs={4} /> <Placeholder xs={6} />{" "}
+							<Placeholder xs={8} />
+						</Placeholder>
+						<Placeholder.Button variant="primary" xs={6} />
+					</Card.Body>
+				</Card>
+			</div>
+		);
+
+	const day = getDay(created_at);
+	const hour = getHour(created_at);
+	return (
+		<div className="container">
+			<Stack gap={3}>
+				<div className="bg-light border">
+					<Card
+						style={{ width: "18rem" }}
+						className="text-dark d-inline"
+						key={id}
+					>
```

Comment:
![image](https://user-images.githubusercontent.com/22796757/199285975-d54d00b7-9d81-4070-8df2-7d2f1fcb644f.png)
you have used react bootstrap here but I think the way this is set out might be a little confusing if the article was very long, all the information would be below the body of the text, IDK if its intentional but it seems a little strange, also what are Second Item and Third Item????

Its also quite div soupy

Reply:
Thanks for the feedback